language/tools/move-mv-llvm-compiler/src/stackless/module_context.rs

```diff
@@ -848,6 +848,84 @@ impl<'mm, 'up> ModuleContext<'mm, 'up> {
         attrs
     }
 
+    // This function extracts an entry function actual arguments from
+    // AccountInfo data field.  The field is a byte array containing
+    // values of actual arguments in sequential order without gaps.
+    fn emit_entry_arguments(
+        &self,
+        fn_env: &mm::FunctionEnv,
+        accounts: &llvm::AnyValue,
+    ) -> Vec<llvm::AnyValue> {
+        if fn_env.get_parameter_count() == 0 {
+            return vec![];
+        }
+        let llcx = self.llvm_cx;
+        let vec_ty = llcx.get_anonymous_struct_type(&[
+            llcx.ptr_type(),
+            llcx.int_type(64),
+            llcx.int_type(64),
+        ]);
```

Comment:
We can use an existing rtty function for this (try to keep rtty type knowledge encapsulated in rttydesc): 
`let vec_ty = self.rtty_cx.get_llvm_type_for_move_native_vector();`

(Likewise for another occurrence in ` emit_rtcall_deserialize` in the `ll_sret = ...` statement. I did not notice this in an earlier patch).

Reply:
I fixed this. Thank you!